python/nano/src/bigdl/nano/automl/pytorch/objective.py

```diff
@@ -183,19 +147,57 @@ def _fix_data_args(self,
                               "or `val_dataloaders` to `trainer.search(datamodule=...)`")
         return train_dataloaders, val_dataloaders, datamodule
 
+    def _auto_optimize(self, model, scores):
+        Score = namedtuple("Score", self.target_metric)
+        best_score = Score(*scores)
+        model.eval()
+
+        # here we suppose nn.model is attached to plmodel by attribute "model"
+        original_model = model.model
+        for optimization in ['openvino', 'onnxruntime', 'jit']:
+            # may enable more optimizations later
+            try:
+                if optimization == 'jit':
+                    optim_model = Trainer.trace(original_model, input_sample=self.input_sample,
+                                                accelerator=optimization,
+                                                use_ipex=True, channels_last=False)
+                else:
+                    optim_model = Trainer.trace(original_model, input_sample=self.input_sample,
+                                                accelerator=optimization)
+            except Exception:
+                # some optimizations may fail, just skip and try next
+                continue
+            model.model = optim_model
+            self.searcher._validate(model, self.val_dataloaders)
+            optim_score = []
+            for metric in self.target_metric:
+                score = self.searcher.trainer.callback_metrics[metric].item()
+                optim_score.append(score)
+            # compare optim_scores with original scores to find a similar
+            # loss value with less latency
+            optim_score = Score(*optim_score)
+            usable = True
```

Comment:
add some comment to explain why this usable is needed (for future developer)

Reply:
done.